pkg/bindings/pods/pods.go

```diff
@@ -98,17 +98,23 @@ func Pause(ctx context.Context, nameOrID string) (*entities.PodPauseReport, erro
 	return &report, response.Process(&report)
 }
 
-// Prune removes all non-running pods in local storage.
-func Prune(ctx context.Context) error {
+// Prune by default removes all non-running pods in local storage.
+// And with force set true removes all pods.
+func Prune(ctx context.Context, force *bool) ([]*entities.PodPruneReport, error) {
```

Comment:
Do we _need_ force to be a pointer? If not, let's make it call by value.

Reply:
_force_ need not be a pointer. However, I noticed we have used pointers in other references of similar use case hence went with maintaining consistency eg: https://github.com/containers/libpod/blob/master/pkg/bindings/pods/pods.go#L155.

Should I still go ahead and change it?

Reply:
Pointers are usually used to find out whether a certain value has been set on the CLI. This allows for distinguishing it from being set (true/false) and not being set (nil), which allows for using defaults in backend code. As we are not doing that in this case, there's no need to have a pointer.

Reply:
Thank you. Wrt today discussion force flag for pod prune is not required here hence marking resolved.